examples/tutorials/analysis-1d/ebl.py

```diff
@@ -0,0 +1,292 @@
+"""
+Account for spectral absorption due to the EBL
+==============================================
+
+Gamma rays emitted from extra-galactic objects, eg blazars, interact
+with the photons of the Extragalactic Background Light (EBL) through
+pair production and are attenuated, thus modifying the intrinsic
+spectrum.
+
+Various models of the EBL are supplied in `GAMMAPY_DATA`. This
+notebook shows how to use these models to correct for this interaction.
+
+"""
+
+######################################################################
+# Setup
+# -----
+#
+# As usual, we’ll start with the standard imports …
+#
+
+from copy import deepcopy
+import astropy.units as u
+from astropy.coordinates import Angle, SkyCoord
+from astropy.time import Time
+from regions import CircleSkyRegion
+import matplotlib.pyplot as plt
+from gammapy.catalog import SourceCatalog4FGL
+from gammapy.datasets import SpectrumDatasetOnOff
+from gammapy.estimators import FluxPointsEstimator
+from gammapy.maps import MapAxis, RegionGeom, WcsGeom
+from gammapy.modeling import Fit
+from gammapy.modeling.models import (
+    EBL_DATA_BUILTIN,
+    EBLAbsorptionNormSpectralModel,
+    GaussianPrior,
+    PowerLawSpectralModel,
+    SkyModel,
+)
+
+######################################################################
+# Load the data
+# -------------
+#
+# We will use 6 observations of the blazars PKS~2155-304 taken in 2008 by
+# H.E.S.S. when it was in a steady state. The data have already been
+# reduced to OGIP format\ `SpectrumDatasetOnOff` following the procedure
+# :doc:`/tutorials/analysis-1d/spectral_analysis` tutorial using a
+# `ReflectedRegions` background estimation. The spectra and IRF from the
+# 6 observations have been stacked together.
+#
+# We will load this dataset as a `SpectrumDatasetOnOff` and proceed with
+# the modeling. You can do a 3D analysis as well.
+#
+
+dataset = SpectrumDatasetOnOff.read(
+    "$GAMMAPY_DATA/PKS2155-steady/pks2155-304_steady.fits.gz"
+)
+
+print(dataset)
+
+
+######################################################################
+# Model the observed spectrum
+# ---------------------------
+#
+# The observed spectrum is already attenuated due to the EBL. Assuming
+# that the intrinsic spectrum is a power law, the observed spectrum is a
+# `CompoundSpectrumModel` given by the product of an EBL model with the
+# intrinsic model.
+#
+
+
+######################################################################
```

Comment:
This section in the docs is a little hard to read so I have suggested some changes to help with that. Specifically adding extra lines between the print statements